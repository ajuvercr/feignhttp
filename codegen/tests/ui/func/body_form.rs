use feignhttp_codegen::post;

// error: request must have only one of body or form
//    |  pub async fn send_post(#[body] b: S, #[form] f: S) -> Result<String> {}
//    |                                 ^^^^^^^^^^^^^^^^^^

struct S;

#[post("http://xxx")]
pub async fn send_post(#[body] b: S, #[form] f: S) -> Result<String> {}

fn main() {}
