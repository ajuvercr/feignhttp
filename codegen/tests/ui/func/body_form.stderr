error: request must have only one of body or form
  --> tests/ui/func/body_form.rs:10:32
   |
10 | pub async fn send_post(#[body] b: S, #[form] f: S) -> Result<String> {}
   |                                ^^^^^^^^^^^^^^^^^^
